src/main/java/data/workouts/WorkoutList.java

```diff
@@ -0,0 +1,44 @@
+package data.workouts;
+
+import data.exercises.ExerciseList;
+import data.exercises.InvalidExerciseException;
+
+import java.util.ArrayList;
+
+public class WorkoutList {
+    ExerciseList exerciseList;
+    ArrayList<Workout> workoutsList = new ArrayList<>();
+
+    public WorkoutList(ExerciseList exerciseList) {
+        this.exerciseList = exerciseList;
+    }
+
+    public Workout createWorkout(String userArgument) throws ArrayIndexOutOfBoundsException,
+            InvalidExerciseException, InvalidWorkoutException {
+        String userExerciseInput = userArgument.split("/reps")[0].trim();
+        String userRepsInputString = userArgument.split("/reps")[1].trim();
+        int userRepsInput = Integer.parseInt(userRepsInputString);
```

Comment:
Should we consider the cases where the userRepsInput is not a number (i.e String) by throwing NumberFormatException?

Reply:
Yup, I intend to work on implementing more checks on the process. Thanks for pointing it out!

Reply:
Added issue #18 to track this feedback.

Reply:
Catching and handling `NumberFormatException` has been implemented in Commit 1742668.

https://github.com/AY2122S2-CS2113T-T09-2/tp/issues/18#issuecomment-1059878336